tensorflow/contrib/layers/python/layers/initializers.py

```diff
@@ -46,7 +46,7 @@ def xavier_initializer(uniform=True, seed=None, dtype=dtypes.float32):
   Args:
     uniform: Whether to use uniform or normal distributed random initialization.
     seed: A Python integer. Used to create random seeds. See
-      [`set_random_seed`](../../api_docs/python/constant_op.md#set_random_seed)
+      [`set_random_seed`](../../../../../api_guides/python/constant_op#random-tensors)
```

Comment:
Yes, we have a doc generator which fills in that token with a link. And I lied to you, you should write:

```prettyprint
@{$constant_op#random-tensors$set_random_seed}
```

This will produce the whole markdown link, including the [] part.

You could also simply write @{set_random_seed}, which will link to that function's docs, which in turn link to the guide. I think that's preferred.

Which should turn into the link you want. We will have public docs on how to write docs soon, I can't wait.